common/lib/xmodule/xmodule/templates/problem/forumularesponse.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
in the sandbox, Math Expression Input is listed twice under Problem > Advanced. Is that correct?
common/lib/xmodule/xmodule/templates/problem/formularesponse.yaml
common/lib/xmodule/xmodule/templates/problem/forumularesponse.yaml
Note that this file name has an extra "u" in formula -- are both needed? they are similar but not identical